### DISCUSSION TITLE
can i get page number with page content?

### DISCUSSION THREAD

**Discussion by mdoulabi1** [spatie/pdf-to-text#73] [Q&A]
is there any way to get page number with its contenet?
for example:
page 1 : full content

**Comment by younus93** [comment]
> is there any way to get page number with its contenet? for example: page 1 : full content

Were you able to get this to work?

**Comment by sandromedina** [comment]
I didn't found a direct way to get the page number but you could use the options:

> -f int: the first page to convert
> -l int: the last page to convert 

to loop through the pages of the file untill the rise of the exception `Spatie\PdfToText\Exceptions\CouldNotExtractText` with a message that contains something like _Wrong page range given: the first page (5) can not be after the last page (4)_ .
